k8s/operators/src/pool/main.rs

```diff
@@ -672,8 +711,9 @@ async fn pool_controller(args: ArgMatches) -> anyhow::Result<()> {
             error!(%error, "Was not able to fully migrate existing v1 CRs");
         });
```

Comment:
Does this need to be moved up or msp migrates to latest version by default?

Reply:
I think should be okay to migrate it to v1beta1 directly. We call migrate_and_clean_msps (migrates to v1beta1) after discarding older v1alpha1 crd. I will test it anyway.